modules/tide_content_collection/json/schema.json

```diff
@@ -257,7 +265,14 @@
                     "type": "string"
                   },
                   "options": {
-                    "type": "object"
+                    "type": "object",
+                    "properties": {
+                      "model": {
+                        "type": "string",
+                        "description": "The URL query key for this field, and a fallback if elasticsearch-field is undefined."
```

Comment:
Nitpick: Is `elasticsearch-field` used as a "word". Perhaps better just as "the Elasticsearch field".

Reply:
The above field references to 
elasticsearch-field 
https://github.com/salsadigitalauorg/content-collection-component/blob/master/src/validation/schema.json#L283